src/build.py

```diff
@@ -783,7 +767,14 @@ def LLVM():
 
   proc.check_call(command, cwd=LLVM_OUT_DIR, env=cc_env)
   proc.check_call(['ninja', '-v'] + jobs, cwd=LLVM_OUT_DIR, env=cc_env)
-  proc.check_call(['ninja', 'check-all'], cwd=LLVM_OUT_DIR, env=cc_env)
+
+  def RunWithUnixUtils(cmd, **kwargs):
+    if IsWindows():
+      return Git(['bash'] + cmd, **kwargs)
+    else:
+      return proc.check_call(cmd, **kwargs)
+
+  RunWithUnixUtils(['ninja', 'check-all'], cwd=LLVM_OUT_DIR, env=cc_env)
```

Comment:
Yeah this is really-local and really-hacky (and I hope to keep it from being used anywhere else and hopefully remove it eventually) so I kind of kept it here for that reason.